src/auth.ts

```diff
@@ -303,8 +297,12 @@ const getAuth = function (
 	 * });
 	 */
 	async function getUserId(): Promise<number> {
-		const { id } = await getUserDetails();
-		return id;
+		const actor = await getActorDetails();
+
+		if (actor.actorType === 'user') {
+			return actor.actorTypeId;
+		}
+		throw new Error(`${actor.actorType} key is not associated to any user`);
```

Comment:
NIT: Let's follow the early return(/throw) pattern.
```suggestion
		if (actor.actorType !== 'user') {
			throw new Error(`The authentication credentials in use are not of a user`);
		}
		return actor.actorTypeId;
```